entity-types/infra-awsmediatailor/dashboard.json

```diff
@@ -0,0 +1,732 @@
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT average(`aws.mediatailor.Avail.Duration`) as 'Total Ad Avail Duration', average(`ws.mediatailor.Avail.ObservedDuration`) as 'Avail Observed Duration', average(`aws.mediatailor.Avail.ObservedFilledDuration`) as 'Avail Observed Filled Duration', average(`aws.mediatailor.Avail.ObservedSlateDuration`) as 'Avail Observed Slate Duration', average(`aws.mediatailor.Avail.FilledDuration`) as 'Avail Filled Duration' TIMESERIES AUTO"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "units": {
+              "unit": "MS"
+            }
+          }
+        },
+        {
+          "title": "Origin manifest file size",
+          "layout": {
+            "column": 5,
+            "row": 5,
+            "width": 4,
+            "height": 2
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT average(aws.mediatailor.Origin.ManifestFileSizeBytes) as 'Origin Manifest File Size' TIMESERIES AUTO"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "units": {
+              "unit": "BYTES"
+            }
+          }
+        },
+        {
+          "title": "Origin latency",
+          "layout": {
+            "column": 9,
+            "row": 5,
+            "width": 4,
+            "height": 2
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT average(`aws.mediatailor.Origin.Latency`) as 'Origin Latency' TIMESERIES AUTO "
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "units": {
+              "unit": "MS"
+            },
+            "yAxisLeft": {
+              "zero": true
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "variables": []
```

Comment:
you have to remove this line

Reply:
@gvassallo, validation failed after doing changes. Could you please check